### DISCUSSION TITLE
What kind of AI does the project use?

### DISCUSSION THREAD

**Discussion by Markg546** [jomjol/AI-on-the-edge-device#729] [Q&A]
I've been wondering what type of AI algorithm was used, considering it has a tflite file.

Thanks in advance!

**Comment by jomjol** [comment]
There is Tensorflow Algo used - Details can be found here:

- Digital digits: https://github.com/jomjol/neural-network-digital-counter-readout
- Analog counter: https://github.com/jomjol/neural-network-analog-needle-readout
